test/test_inference.py

```diff
@@ -30,7 +30,7 @@
 import numpy
 from pycbc import distributions
 from pycbc.inference import likelihood
-from pycbc.inference import prior
+from pycbc.distributions import JointDistribution
```

Comment:
Its importing ``from pycbc import distributions`` a couple lines above. This could just be removed and changed the call below to be ``distributions.JointDistribution``.